Features/ErrorViews/ErrorView.swift

```diff
@@ -0,0 +1,106 @@
+//
+//  ErrorView.swift
+//  Movies
+//
+//  Created by Diana Chizhik on 18.07.22.
+//
+
+import UIKit
+
+protocol ErrorViewDelegate: AnyObject {
+    func dismissController()
```

Comment:
I wasn't being picky on this but the time has come 😄 It's recommended to pass the object as the first parameter of delegates, usually ommiting the name. In this case this would be:
```suggestion
    func dismissController(_ errorView: ErrorView)
```
Later, it would be called as:
```swift
delegate?.dismissController(self)
```